bin/deploy.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PUBLISH=0
+CHEF_VER="latest"
```

Comment:
Right now, there is no way to set it through the pipeline as it accepts  `payload` as the only parameter. I think it would be good if we remove that switch from `build_image.sh`, `deploy.sh` and handle it plainly through the template's variables section where a default can be set.